binary/src/Cardano/Binary/Serialize.hs

```diff
@@ -60,17 +66,13 @@ serializeBuilder = CBOR.Write.toBuilder . toCBOR
 --   The output is represented as an 'LByteString' and is constructed
 --   incrementally.
 serializeEncoding :: Encoding -> BSL.ByteString
-serializeEncoding =
-  Builder.toLazyByteStringWith strategy mempty . CBOR.Write.toBuilder
-  where
-    -- 1024 is the size of the first buffer, 4096 is the size of subsequent
-    -- buffers. Chosen because they seem to give good performance. They are not
-    -- sacred.
-        strategy = Builder.safeStrategy 1024 4096
+serializeEncoding = serialize
+{-# DEPRECATED serializeEncoding "Use `serialize` instead, since `Encoding` has `EncCBOR` instance" #-}
```

Comment:
`EncCBOR` or `ToCBOR` ?

Reply:
Thank you. Another good catch. This snuck in due to some other refactoring/renaming that is underway.